src/core/BUILD

```diff
@@ -4178,6 +4178,15 @@ grpc_cc_library(
     deps = ["//:gpr_platform"],
 )
 
+grpc_cc_library(
+    name = "xds_enabled_server",
+    hdrs = [
+        "ext/xds/xds_enabled_server.h",
+    ],
+    language = "c++",
+    deps = ["//:gpr_platform"],
```

Comment:
If you remove the include as per my other comment, then I think we don't think we need this dependency.

Reply:
Done